android-vjestina-homework-movie-app-data-mock/app/src/main/java/agency/five/codebase/android/movieapp/ui/component/ActorCard.kt

```diff
@@ -0,0 +1,89 @@
+package agency.five.codebase.android.movieapp.ui.component
+
+import agency.five.codebase.android.movieapp.R
+import agency.five.codebase.android.movieapp.mock.MoviesMock
+import agency.five.codebase.android.movieapp.ui.theme.Shapes
+import agency.five.codebase.android.movieapp.ui.theme.Typography
+import agency.five.codebase.android.movieapp.ui.theme.spacing
+import androidx.compose.foundation.layout.Column
+import androidx.compose.foundation.layout.height
+import androidx.compose.foundation.layout.padding
+import androidx.compose.foundation.layout.width
+import androidx.compose.material.Card
+import androidx.compose.material.MaterialTheme
+import androidx.compose.material.Text
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.layout.ContentScale
+import androidx.compose.ui.res.dimensionResource
+import androidx.compose.ui.text.style.TextOverflow
+import androidx.compose.ui.tooling.preview.Preview
+import coil.compose.AsyncImage
+
+
+data class ActorCardViewState(
+    val imageUrl: String,
+    val name: String,
+    val character: String,
+)
+
+@Composable
+fun ActorCard(
+    modifier: Modifier = Modifier,
+    actorCardViewState: ActorCardViewState
```

Comment:
It is a convention that mandatory parameters (without default value) should be before optional parameters (with default values).
This makes it possible to call the function without explicitly adding parameter names when you are not passing optional parameter, for example if I want to call ActorCard only passing the ViewState
Your case: `ActorCard(actorCardViewState = actorCardViewState)`
With proper parameter ordering: `ActorCard(actorCardViewState)`

Please fix this for all your functions.